app/src/main/java/de/langerhans/odintools/main/MainActivity.kt

```diff
@@ -205,6 +227,25 @@ fun SettingsScreen(viewModel: MainViewModel = hiltViewModel(), navigateToOverrid
                 ) {
                     viewModel.updateVibrationPreference(it)
                 }
+                SettingsHeader(R.string.battery)
+                SwitchableTriggerPreference(
+                    icon = R.drawable.ic_electrical_services,
+                    title = R.string.chargeLimit,
+                    description = R.string.chargeLimitDescription,
+                    state = uiState.chargeLimitEnabled,
+                    onClick = { viewModel.chargeLimitClicked() },
+                ) {
+                    viewModel.updateChargeLimitPreference(it)
+                }
+            }
+            if (BuildConfig.DEBUG) {
```

Comment:
Usually people wouldn't run debug builds no?

Reply:
This is something I've been debating with myself a lot. I think I was looking for a reason to add this feature, which serves a very niche purpose on its own. As far as it goes, people usually wouldn't run debug builds. Please let me know if this is something that isn't very useful in its current state.

Reply:
I didn't mean to say it's not useful cause it certainly is for cases where logs can help. Was just wondering if we should expose it for release builds as well :)

Reply:
Resolved in 884c8dc